CHANGELOG.md

```diff
@@ -27,12 +28,60 @@ fn update(&mut self, _ctx: &mut Context) -> GameResult {
 ```
 This will print `Coroutine says: "I came from a coroutine!"` after 100 frames.
 
+### 3d
+
+You can now do 3d rendering with bevy. This entails a lot of new api ground to cover. Here are the main points.
+There is now a `Canvas3d` and `Drawable3d`. These are used to do any 3d rendering instead of `Canvas` a simple example of drawing may look like this:
+```rust
+fn draw(&mut self, ctx: &mut Context) -> GameResult {
+  let mut canvas3d = Canvas3d::from_frame(ctx, &mut self.camera, Color::BLACK);
+  canvas3d.draw(ctx, &drawable, draw_param);
+  canvas3d.finish(ctx)?;
+}
+```
+
+Some types that implement Drawable3d by default are `Mesh3d` and `Model`. `Mesh3d` is used for a singular mesh.
+`Model` us used for more complicated objects that may be made up of multiple meshes. `Model` also provides a way to load basic gltf/glb files and obj files.
+(Note not nearly all of gltf is supported and obj models must be triangulated in your software of choice and no material files will be read).
```

Comment:
> (Note: not all of gltf is supported. obj models must be triangulated in your software of choice, and no material files will be read).

Reply:
Thanks for both your comments i'll change them up after the rewrite i'm doing atm I found a pretty large issue that needs addressing